README.md

```diff
@@ -423,24 +411,31 @@ To solve this error You must go to [hoyolab.com](https://www.hoyolab.com/genshin
 Every endpoint in mihoyo's api requires authentication, this is in the form of a cookie and an authkey.
 User stats use a cookie and wish history uses an authkey.
 
-The cookie is bound to the user and as far as I know can only be reset by changing your password, so remember to never give your cookie to anyone. For extra safety you may want to create an alt account, so your real account is never in any danger. This token will allow you to view public stats of all users and private stats of yourself.
+The cookie is bound to the user and as far as I know, it can only be reset by changing your password. Safety wise, these cookies only give an access to your own statistics and allow few actions (claim daily reward) so it isn't dangerous by itself, but you should still keep it safe, as Mihoyo's API could include more sensitive features in the future. For extra safety you may want to create an alt account, so your real account is never in any danger. This alt token will allow you to view public stats of all users and private stats of yourself.
 
 The authkey is a temporary token to access your wish history. It's unique for every user and is reset after 24 hours. It cannot be used to view the history of anyone else. It is fine to share this key with anyone you want, the only "private" data they will have access to is the wish history.
 
 Tip for developers: the first 682 characters (85 bytes) of the authkey are always the same for each user. You can use this to easily indentify if an authkey belongs to the same user as another authkey.
 
+Element | Access level | Sensitivity
+|--|--|--|
+`UID` | Read access to your Battle Chronicle's public data | Safe
+`ltuid` + `ltoken` | Read access to your full Battle Chronicle (characters, abyss clears, ...) + few actions possible (claim daily reward) | Quite safe
```

Comment:
definitely not just "quite safe". "few actions" skips over quite a few dangerous things

Reply:
Actually, I don't see anything else you can do with `ltuid` and `ltoken` apart claiming daily rewards and changing the account's public visibility. You can't be connected to the Hoyolab account nor post in his stead.
Did you think of something else?

Reply:
I have tested this once again just to be sure and you absolutely can use those to work with the hoyolab account through the API. This is not a matter of logging-in in the browser.